pkgs/by-name/me/menulibre/package.nix

```diff
@@ -0,0 +1,64 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+, gnome-menus
+, gtk3
+, intltool
+, gobject-introspection
+, wrapGAppsHook
+, testers
+, menulibre
+}:
+
+python3Packages.buildPythonPackage rec {
+  name = "menulibre";
+  version = "2.2.3";
+
+  src = fetchFromGitHub {
+    owner = "bluesabre";
+    repo = "menulibre";
+    rev = "menulibre-${version}";
+    hash = "sha256-E0ukq3q4YaakOI2mDs3dh0ncZX/dqspCA+97r3JwWyA=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    pygobject3
+    gnome-menus
+    psutil
+    distutils-extra
+  ];
+
+  nativeBuildInputs = [
+    gtk3
+    intltool
+    gobject-introspection
+    wrapGAppsHook
+  ];
+
+  postPatch = ''
+    sed -i "/__menulibre_data_directory__/s@(data_dir)@'$out/share/menulibre'@" setup.py
+  '';
+
+  preBuild = ''
+    export HOME=$TMPDIR
+  '';
+
+  postFixup = ''
+    sed -i "/Exec=/c\Exec=$out/bin/menulibre" \
```

Comment:
same here

Reply:
I managed to not need to patch the desktop file, but what if I had to? Is there some way to represent the `sed` command used here using `substituteInPlace`? As in find some text and replace the entire line that contains it with some other text.

Reply:
you can still use sed